### PR TITLE
Add itesm->items and another suggestion to iterm

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8950,6 +8950,10 @@ eliminetaion->elimination
 elimintate->eliminate
 eliminte->eliminate
 elimnated->eliminated
+elipse->ellipse, eclipse,
+elipses->eclipses, ellipsis,
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: This could also be a typo of ellipses

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8950,6 +8950,10 @@ eliminetaion->elimination
...
+elipsis->ellipsis, eclipses,
+elipsises->ellipsis
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: This could also be a typo of ellipses

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**